interval.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+)
+
+type RefreshInterval time.Duration
+
+func (self RefreshInterval) String() string {
+	return time.Duration(self).String()
+}
+
+func (self *RefreshInterval) UnmarshalYAML(
+	unmarshal func(interface{}) error,
+) error {
+	i := 0
+	if err := unmarshal(&i); err == nil {
+		*self = RefreshInterval(time.Duration(i) * time.Second)
+		return nil
+	}
+	s := ""
+	if err := unmarshal(&s); err == nil {
+		i, err := strconv.ParseUint(s, 10, 64)
+		if err == nil {
+			*self = RefreshInterval(time.Duration(i) * time.Second)
+			return nil
```

Comment:
What do you want to do here? For example, we can silently reset the value to `MIN_REFRESH_INTERVAL` or we can reset and print a warning. We can also return an error.